packages/block-editor/src/components/global-styles/style.scss

```diff
@@ -73,18 +73,67 @@
 
 
 .block-editor-global-styles-background-panel__inspector-media-replace-container {
-	border: 1px solid $gray-300;
-	border-radius: 2px;
 	// Full width. ToolsPanel lays out children in a grid.
 	grid-column: 1 / -1;
 
+	& > div {
+		// Border styles.
+		border-left: 1px solid $gray-300;
+		border-right: 1px solid $gray-300;
+		border-bottom: 1px solid $gray-300;
+
+		&:not(:only-child):nth-child(1 of &) {
+			border-top-left-radius: $radius-block-ui;
```

Comment:
Might want to use `$radius-small` instead - we're aiming to deprecate `$radius-block-ui`, see #64007

Reply:
Thanks for the feedback, I have addressed it